scripts/install_deps.sh

```diff
@@ -300,7 +300,8 @@ case $(uname -s) in
                         install_z3="libz3-dev"
                         ;;
                     bionic)
-                        echo "Installing solidity dependencies on Ubuntu Bionic (18.04)."
+                    focal)
+                        echo "Installing solidity dependencies."
                         install_z3="libz3-dev"
```

Comment:
This one won't work, that'd be the stock ubuntu libz3 which is too old... Actually I'm generally not sure this entire script works all that well these days...

Reply:
Actually wrong - focal actually has the right one I just noticed! It won't work for bionic above, though :-D.